Explorer/Assets/DCL/SDKComponents/AudioStream/AudioStreamComponent.cs

```diff
@@ -0,0 +1,84 @@
+﻿using DCL.ECSComponents;
+using DCL.Optimization.Pools;
+using DCL.Utilities.Extensions;
+using RenderHeads.Media.AVProVideo;
+using System;
+using System.Runtime.CompilerServices;
+
+namespace DCL.SDKComponents.AudioStream
+{
+    public struct AudioStreamComponent : IPoolableComponentProvider<MediaPlayer>
+    {
+        private const float DEFAULT_VOLUME = 1f;
+        private static IComponentPool<MediaPlayer> mediaPlayerPool;
+
+        private string url;
+
+        private MediaPlayer mediaPlayer { get; }
+
+        MediaPlayer IPoolableComponentProvider<MediaPlayer>.PoolableComponent => mediaPlayer;
+
+        Type IPoolableComponentProvider<MediaPlayer>.PoolableComponentType => typeof(MediaPlayer);
+
+        public AudioStreamComponent(PBAudioStream sdkComponent, IComponentPoolsRegistry poolsRegistry, bool isCurrentScene)
+        {
+            mediaPlayerPool ??= poolsRegistry.GetReferenceTypePool<MediaPlayer>();
```

Comment:
Just resolve this pool once in the system, static field is a bad idea

Reply:
Nice one! Fully agree. I brought it there when I was searching for proper isolation of the required packages (as a temporary hack). But now it should work fine just inside the system (as in other similar systems).